docs/api/hardhat/getting-started.md

```diff
@@ -2,28 +2,27 @@
 
 [Hardhat](https://hardhat.org) is an Ethereum development environment, designed for easy smart contract development in Solidity. One of its most prominent features is extendability: you can easily add new plugins to your hardhat project.
 
-zkSync has three plugins for hardhat:
+zkSync has three plugins for Hardhat:
 
-- `@matterlabs/hardhat-zksync-solc` for smart contract compilation using Solidity.
-- `@matterlabs/hardhat-zksync-vyper` for smart contract compilation using Vyper.
-- `@matterlabs/hardhat-zksync-deploy` for smart contract deployment.
+- [@matterlabs/hardhat-zksync-solc](./plugins.md#matterlabs-hardhat-zksync-solc) for smart contract compilation using Solidity.
```

Comment:
Should we actually write "for compiling smart contracts written I'm solidity"? And same for vyper

Reply:
fixed in e28f4343571352dec4b23cfb303616a2f8f497fa